### PR TITLE
feat: add top tvl pairs component

### LINKED ISSUE #103: Add top TVL pairs
Add top TVL pairs component on the pair list page.

image

### PR DESCRIPTION
**Pull request by guerrap**
# Summary

Fixes #103 

Add the `Top TVL Pairs` components, with the first 20 pairs, sorted by `tvl`.
Add internal redirect on `onClick` events for tokens and pairs cards.

image

### CODE REVIEW

File: src/components/CarrotBadge/styled.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 3px;
+
+  padding: 2px 3px;
+
```
Comment (wixzi) [inline src/components/CarrotBadge/styled.jsx:10]: may be remove all unwanted spaces

Comment (wixzi) [inline src/components/CarrotBadge/styled.jsx:10]: Did you add these spaces intentionally. Else can be removed.

**Review by wixzi — APPROVED** [review]
Looks good. Great job @guerrap

**Comment by wixzi** [comment]
Is it as per design?
Screenshot 2022-09-09 at 22 57 14

**Comment by guerrap** [comment]
> Is it as per design? Screenshot 2022-09-09 at 22 57 14

Yes, the empty space is where the `farming` and `carrot` badges show up if active, maybe we can still add the badges, but grayed if disabled, so that the space is not empty.

**Comment by guerrap** [comment]
Add disabled `farming` and `carrot` badges to the pair cards.

image

**Review by wixzi — APPROVED** [review]

**Pull request merged**